src/smite/smiteAPI.js

```diff
@@ -170,6 +170,20 @@ class Smite {
     })
   }
 
+  getGodLeaderBoard (godId, queue) {
+    let url = util.genUrl(this.smiteUrl, 'getgotleaderboard', this.devId, this.authKey, process.env[`SMITE_${this.platform}_SESSION`]) + '/' + godId +
```

Comment:
I just occurred to me that I should probably put some contribution guidelines in somewhere :P